grails-app/domain/au/org/emii/portal/Server.groovy

```diff
@@ -223,4 +222,10 @@ class Server {
             operations << operation
         }
     }
+
+    def getUrlListDownloadSubstitutions() {
+        return grailsApplication.config.knownServers.find {
+            this.uri.startsWith(it.uri)
+        }?.urlListDownloadSubstitutions ?: [:]
```

Comment:
Note that it's still necessary to have the `?` after find - since there's no guarantee that the server will be in the known servers config.  I think this is slightly complicated by the fact that as it stands currently, *some* server config is coming from the DB, some from the config file.

When everything is coming from the config file, then it would probably be impossible to get to this method unless the server had config... if that makes sense.